app/views/vacancies/index.html.slim

```diff
@@ -9,7 +9,8 @@
 h1 class="govuk-heading-l" role="heading" aria-level="1"
   = t("jobs.search_result_heading", count: number_with_delimiter(@vacancies_search.total_count))
 
-- if @vacancies_search.active_criteria? && @vacancies.count.positive?
+- any_vacancies = @vacancies.any?
```

Comment:
memoising the paginated results count saves us multiple DB queries. I believe we could even use the already memoised `@vacancies_search.total_count` here and not trigger a new single query.
TODO: Check this.